tests/e2e/taqueria-plugin-metadata.spec.ts

```diff
@@ -1,208 +1,154 @@
+			'name': 'test-project-name',
+			'projectDescription': 'test-project-description',
+			'authors': [
+				'KentBeck',
+				'MartinFowler',
+				'ErichGamma',
+			],
+			'homepage': 'http://taqueria.io',
+			'license': '007',
 		});
-	};
-
-	test.skip('metadata plugin should create a contract metadata.json file', async () => {
-		await runCliWithPrompts(`generate-metadata hello-tacos`, [
-			['name', 'test-name'],
-			['description', 'test-description'],
-			['author', 'test-author'],
-			['url', 'test-url'],
-			['license', 'test-license'],
-		]);
-
-		const metadataFileContents = await fsPromises.readFile(`${taqueriaProjectPath}/artifacts/hello-tacos.json`, {
-			encoding: 'utf-8',
-		});
-		expect(metadataFileContents).toMatch(/name.*test-name/i);
-		expect(metadataFileContents).toMatch(/description.*test-description/i);
-		expect(metadataFileContents).toMatch(/authors(.|\n)*test-author/i);
-		expect(metadataFileContents).toMatch(/homepage.*test-url/i);
-		expect(metadataFileContents).toMatch(/license.*test-license/i);
-	});
 
-	test.skip('metadata plugin should re-create a contract metadata.json using existing values', async () => {
-		await runCliWithPrompts(`generate-metadata hello-tacos`, [
-			['name', 'test-name'],
-			['description', 'test-description'],
-			['author', 'test-author'],
-			['url', 'test-url'],
-			['license', 'test-license'],
-		]);
-
-		await runCliWithPrompts(`generate-metadata hello-tacos`, [
-			['name', ''],
-			['description', ''],
-			['author', ''],
-			['url', ''],
-			['license', ''],
-		]);
-
-		const metadataFileContents = await fsPromises.readFile(`${taqueriaProjectPath}/artifacts/hello-tacos.json`, {
-			encoding: 'utf-8',
-		});
-		expect(metadataFileContents).toMatch(/name.*test-name/i);
-		expect(metadataFileContents).toMatch(/description.*test-description/i);
-		expect(metadataFileContents).toMatch(/authors(.|\n)*test-author/i);
-		expect(metadataFileContents).toMatch(/homepage.*test-url/i);
-		expect(metadataFileContents).toMatch(/license.*test-license/i);
+		await cleanup();
 	});
 
-	test.skip('metadata plugin should ask for contract name if not provided', async () => {
-		await runCliWithPrompts(`generate-metadata`, [
-			['contract', 'hello-tacos'],
-			['name', 'test-name'],
-			['description', 'test-description'],
-			['author', 'test-author'],
-			['url', 'test-url'],
-			['license', 'test-license'],
-		]);
-
-		const metadataFileContents = await fsPromises.readFile(`${taqueriaProjectPath}/artifacts/hello-tacos.json`, {
-			encoding: 'utf-8',
-		});
-		expect(metadataFileContents).toMatch(/name.*test-name/i);
-		expect(metadataFileContents).toMatch(/description.*test-description/i);
-		expect(metadataFileContents).toMatch(/authors(.|\n)*test-author/i);
-		expect(metadataFileContents).toMatch(/homepage.*test-url/i);
-		expect(metadataFileContents).toMatch(/license.*test-license/i);
-	});
+	test.skip('generate-metadata will error if no contract name provided', async () => {
```

Comment:
Is this skip intentional?

Reply:
Any test marked skipped will be reviewed and repaired in turn.